components/grid/export/pdf.md

```diff
@@ -10,17 +10,223 @@ position: 1
 
 # PDF Export
 
-The [Telerik Document Processing tools](slug://dpl-in-blazor) that come with your Blazor license let you generate a PDF file based on the data in the grid.
+You can export the Grid to PDF with the click of a button. The current filter, sort, page, grouping, column order, and column size are applied to the exported PDF document.
 
-The following sample projects show two ways to implement a PDF export
+When you click the Export button, your browser will receive the resulting file.
+
+#### In This Article
+
+  - [Basics](#basics)
+  - [Programmatic Export](#programmatic-export)
+  - [Customization](#customization)
+  - [Notes](#notes)
+  - [Custom Export](#custom-export)
+
+## Basics
+
+To enable the Grid PDF Export, add a [command button](slug:components/grid/columns/command) with the `PdfExport` command name to the [Grid toolbar](slug:components/grid/features/toolbar).
+
+````RAZOR.skip-repl
+<GridToolBarTemplate>
```

Comment:
You can also show how to do this with the new toolbar syntax.

Reply:
In an offline discussion we've agreed to do that at a later stage for all examples: https://github.com/telerik/blazor/issues/10894